### PR TITLE
#171536528 manager can approve request

### PR DESCRIPTION
**Pull request by izzett222**
#### What does this PR do?
the feature allows managers to approve requests from their direct report
#### Description of Task to be completed?
- add tests for the feature
- add the approve request controllers and route
- add internalization to the feature
- document the feature using swagger
#### How should this be manually tested?
- clone this repo
- setup database according to the readme
- login as manager 
- make a patch request to /api/v1/trips/:requestId/approve
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#171536528
#### Screenshots (if appropriate)
Screen Shot 2020-03-04 at 16 22 44
Screen Shot 2020-03-04 at 16 23 18

#### Questions:

### CODE REVIEW

File: src/tests/unitTests/requestSearchService.test.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+//   it('should return an array of requests found on search', async () => {
+//     field = {
+//       id: 't1e74db7-h610-4f50-9f45-e2371j331ld5'
+//     };
+//     const requests = await TripsService.searchRequest(field, user);
+//     expect(requests).to.be.an('array');
+//     expect(requests[0].dataValues.managerId).to.equal('75c34027-a2f0-4b50-808e-0c0169fb074w');
+//     expect(requests[0].dataValues.id).to.equal('t1e74db7-h610-4f50-9f45-e2371j331ld5');
+//   });
+// });
```
Comment (Ntare22) [inline src/tests/unitTests/requestSearchService.test.js]: Why is this code commented?

**Review by Ntare22 — COMMENTED** [review]
Some code is commented, what happens when uncommented!

File: src/tests/unitTests/requestSearchService.test.js (hunk 1 above)
Comment (izzett222) [reply src/tests/unitTests/requestSearchService.test.js]: I forgot to remove it when I refactored my tests

Comment (izzett222) [reply src/tests/unitTests/requestSearchService.test.js]: I removed them

**Review by Mnickii — APPROVED** [review]
LGTM

**Review by Baraka-Mugisha — APPROVED** [review]

**Review by Ntare22 — APPROVED** [review]

**Pull request merged**